<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controls</class>
 <widget class="QWidget" name="controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visible Controls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="openCheckBox">
     <property name="text">
      <string>Open file button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="playPauseCheckBox">
     <property name="text">
      <string>Play / pause button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stopCheckBox">
     <property name="text">
      <string>Stop button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="playPreviousCheckBox">
     <property name="text">
      <string>Play previous button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="playNextCheckBox">
     <property name="text">
      <string>Play next button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="positionCheckBox">
     <property name="text">
      <string>Position slider</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="volumeCheckBox">
     <property name="text">
      <string>Toggle volume control button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="playlistCheckBox">
     <property name="text">
      <string>Toggle playlist button</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fullScreenCheckBox">
     <property name="text">
      <string>Toggle full screen button</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
